AirCasting/Settings/BackendSettingsView.swift

```diff
@@ -53,6 +54,11 @@ struct BackendSettingsView: View {
         Button {
             urlProvider.baseAppURL = url ?? URL(string: "http://aircasting.org/api")!
             presentationMode.wrappedValue.dismiss()
+            do {
+                try logoutController.logout()
+            } catch {
+                Log.info("Error when loging out")
```

Comment:
We need this log , but let's also log this error so we can see why it failed. 
But I think we also need an alert here to inform the user that logging out failed (sorry I didn't make it clear how to handle this error, my bad)